src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -115,6 +115,32 @@ def load_arguments(self, _):
         c.argument('conflict_resolution_policy', options_list=['--conflict-resolution-policy', '-c'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Conflict Resolution Policy, you can enter it as a string or as a file, e.g., --conflict-resolution-policy @policy-file.json or ' + SQL_GREMLIN_CONFLICT_RESOLUTION_POLICY_EXAMPLE)
         c.argument('throughput', help='The throughput of SQL container (RU/s). Default value is 400')
 
+# SQL stored procedure
+    with self.argument_context('cosmosdb sql stored_procedure') as c:
+        c.argument('account_name', options_list=['--account-name', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database-name', '-d'], help="Database name")
+        c.argument('container_name', options_list=['--container-name', '-c'], help="Container name")
```

Comment:
These parameters are used several times. Please use CLIArgumentType to abstract them and reuse the argtype for your parameters. e.g.  https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/_params.py#L48
https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/_params.py#L120